meson.build

```diff
@@ -0,0 +1,180 @@
+# SPDX-FileCopyrightText: 2022 Aetf <aetf@unlimited-code.works>
+#
+# SPDX-License-Identifier: MIT
+
+project('kmscon', 'c',
+  version: '9.0.0',
+  license: 'MIT',
+  # meson 0.58: f-string
+  # meson 0.62: dependency libdl
+  meson_version: '>=0.62.0',
+  default_options: [
+    'warning_level=1',
+    'werror=true',
+    'buildtype=debugoptimized',
+    'c_std=gnu99',
+  ]
+)
+
+add_project_arguments(
+  '-ffast-math',
+  '-fno-strict-aliasing',
+  '-ffunction-sections',
+  '-fdata-sections',
+  '-fstack-protector',
+  '-fvisibility=hidden',
+  '-D_GNU_SOURCE',
+  '-D_POSIX_C_SOURCE=200809L',
+  language: 'c'
+)
+
+#
+# Directory structure
+#
+prefix = get_option('prefix')
+bindir = get_option('bindir')
+sysconfdir = get_option('sysconfdir') / meson.project_name()
+libexecdir = get_option('libexecdir') / meson.project_name()
+mandir = get_option('mandir')
+moduledir = get_option('libdir') / meson.project_name()
+
+#
+# Required dependencies
+#
+python = import('python').find_installation()
```

Comment:
You only use this once, and that is as a command for a generator. The python module provides an installation object suitable for compiling python C-API extensions and installing them / *.py files to the python site-packages directory. You don't need it here, and you don't need to run the python module's complicated distutils/sysconfig scraper script.

```meson
find_program('python3')
```
will work just as well. :)